### ISSUE TITLE
Linking a finding to an existing JIRA issue throws 500 error

### ISSUE THREAD

**Issue by madchap** [DefectDojo/django-DefectDojo#3352]
**Bug description**
```
ERROR:django.request:Internal Server Error: /finding/1077093/edit
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./dojo/user/helper.py", line 77, in _wrapped
    return view_func(request, lookup_value, *args, **kwargs)
  File "./dojo/finding/views.py", line 676, in edit_finding
    if form.is_valid() and (jform is None or jform.is_valid()):
  File "/usr/local/lib/python3.6/site-packages/django/forms/forms.py", line 185, in is_valid
    return self.is_bound and not self.errors
  File "/usr/local/lib/python3.6/site-packages/django/forms/forms.py", line 180, in errors
    self.full_clean()
  File "/usr/local/lib/python3.6/site-packages/django/forms/forms.py", line 382, in full_clean
    self._clean_form()
  File "/usr/local/lib/python3.6/site-packages/django/forms/forms.py", line 409, in _clean_form
    cleaned_data = self.clean()
  File "./dojo/forms.py", line 2185, in clean
    jira_issue_new = jira_helper.jira_get_issue(jira_project, jira_issue_key_new)
  File "./dojo/jira_link/helper.py", line 827, in jira_get_issue
    jira_instance = jira_project.jira_instance
AttributeError: 'NoneType' object has no attribute 'jira_instance'
```

**Steps to reproduce**
1. Edit a finding that has no jira issue linked yet, then push to jira --> jira gets created
2. Re-edit the finding, and try to modify the JIRA ticket value
3. 500 error

**Expected behavior**
The value gets updated.

**Deployment method** *(select with an `X`)*
- [x] Kubernetes
- [ ] Docker
- [ ] setup.bash / legacy-setup.bash

**Environment information**
1.10.0

**Comment by madchap** [comment]
Also happens even when I have a JIRA configuration at the engagement level.

**Comment by valentijnscholten** [comment]
fixed in release/1.10.1
